Tasks/GulpV0/task.json

```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 166,
-        "Patch": 2
+        "Minor": 181,
```

Comment:
Also: this task seems to be due for [sprint 182](https://github.com/microsoft/build-task-team/milestone/7); could you please fix this in task.json and task.loc.json?

Reply:
Bumped task version